src/main/java/com/fasterxml/jackson/core/StreamReadConstraints.java

```diff
@@ -12,6 +12,11 @@
 {
     private static final long serialVersionUID = 1L;
 
+    /**
+     * Default setting for maximum depth: see {@link Builder#maxDepth(int)} for details.
+     */
+    public static final int DEFAULT_MAX_DEPTH = 1000;
```

Comment:
Yes, I was thinking of 1_000 as the max as well. Reading/writing may default to same value.